[2021-05-25 20:56:36,215] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:25:53.192497+00:00 [queued]>
[2021-05-25 20:56:36,452] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:25:53.192497+00:00 [queued]>
[2021-05-25 20:56:36,459] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:36,462] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 20:56:36,463] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:36,521] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T20:25:53.192497+00:00
[2021-05-25 20:56:36,542] {standard_task_runner.py:52} INFO - Started process 179 to run task
[2021-05-25 20:56:36,561] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T20:25:53.192497+00:00', '--job-id', '109', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmp93zc1q97', '--error-file', '/tmp/tmpzo18ajgf']
[2021-05-25 20:56:36,573] {standard_task_runner.py:77} INFO - Job 109: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 20:56:36,828] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:25:53.192497+00:00 [running]> on host 4e0af5d83701
[2021-05-25 20:56:37,167] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T20:25:53.192497+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-25T20:25:53.192497+00:00
[2021-05-25 21:38:08,898] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 21:38:08,973] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 179
[2021-05-25 21:38:08,974] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
